### ISSUE TITLE
`--filter "precursorRefine"` does not work

### ISSUE THREAD

**Issue by glormph** [ProteoWizard/container#8]
First off, great to have msconvert in a docker, I'd love to remove some windows automation here, so thank you!

I have been able to successfully run the container (latest, 44306d4e8420) to convert Thermo raw data to mzML. But, it fails when I specify `--filter "precursorRefine"`, when it exits with
```
[SpectrumWorkerThreads::work] unknown exception in worker thread
```
Not really sure what causes it. Is this an issue with my data/Ubuntu setup only?

**Comment by chambm** [comment]
Hi @glormph, sorry I didn't see this issue. Do you also get this problem on Windows? The precursorRefine and Recalculator filters can be a bit finicky about what kinds of data they accept.

**Comment by glormph** [comment]
Hi Matt, no problem, I think this solved itself, when a newer container version was available, which I'm currently using (3.0.20066) without issues. 

I'm not sure which version I tried which didnt work (likely the container before 3.0.20066), the version string above seems copied from `docker images` or something. On windows (no container) I have used 3.0.19127 which worked fine.

**Comment by sneumann** [comment]
So I guess @glormph could close the issue :-) Yours, Steffen
